src/main/java/com/park/easyrecruit/servlet/application/ApplicationComments.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.park.easyrecruit.servlet.application;
+
+import com.park.easyrecruit.common.*;
+import com.park.easyrecruit.ejb.*;
+import java.io.IOException;
+import java.io.PrintWriter;
+import javax.inject.Inject;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.*;
+import javax.servlet.http.*;
+
+/**
+ *
+ * @author andrei
+ */
+@WebServlet(name = "ApplicationComments", urlPatterns = {"/ApplicationComments"})
+@ServletSecurity(value = @HttpConstraint(rolesAllowed = {"ClientRole" /* ... */}))
```

Comment:
allow also recruiter role, departmentdirector role and ceo. Better use ManageCommentsRole (already defined) or define another

Reply:
Thanks for the tip. Role checks will be added today in a different PR.